Apps/WebClient/src/ClientApp/app/views/timeline.vue

```diff
@@ -473,6 +486,14 @@ export default class TimelineComponent extends Vue {
       });
   }
 
+  private onCovidSubmit() {
```

Comment:
What do you envision these will be useful for?

Reply:
Opening up the new tab with the results page I believe.